### DISCUSSION TITLE
SOC: Schedule auto reboot after kernel updates

### DISCUSSION THREAD

**Discussion by bab-mkedziora** [Security-Onion-Solutions/securityonion#13379] [Ideas]
Hi Security Onion Team,

it would be very helpful if there was an option in SOC to enable and schedule automatic reboots when necessary, e.g. after kernel updates. Like the auto OS update option.

Would it be possible to implement such feature?

Best Regards
Martin

**Comment by dougburks** [comment]
Please see https://github.com/Security-Onion-Solutions/securityonion/discussions/13397.
